README.md

```diff
@@ -43,6 +44,20 @@ This boilerplate comes with batteries included, you’ll find:
 4. Create a new Git repository (`git init`)
 5. Create the initial Git commit (`git commit -a -m "Initial commit"`)
 
+### Telemetry UI
+
+:warning: Metrics are publicly exposed but in your project you should protect it under authorization.
+See https://github.com/mirego/telemetry_ui#security
+
+```elixir
+ # module ElixirBoilerplateWeb.Router
+ scope "/" do
+   pipe_through(:browser)
+   # Metrics can contains sensitive data you should protect it under authorization
+   get("/metrics", TelemetryUI.Web, [], assigns: %{telemetry_ui_allowed: true})
+ end
+```
+
```

Comment:
I don’t think we need this part in the "boilerplate README".

Reply:
We should actually document Telemetry UI here: https://github.com/mirego/elixir-boilerplate/blob/main/BOILERPLATE_README.md#-troubleshooting (and its french counterpart).

Reply:
```
### Metrics

The project exposes a `GET /metrics` route. Dashboard display following metrics :
* HTTP requests
* Ecto
* GraphQL
* Absinthe 
```

Or better suggestion ?

Reply:
I would stay a bit more generic:

```md
### Metrics

The project exposes a [Telemetry UI](https://github.com/mirego/telemetry_ui) dashboard through the `GET /metrics` route that metrics configured [here](lib/elixir_boilerplate/telemetry_ui/telemetry_ui.ex).
```